packages/backend-function/src/function_env_type_generator.ts

```diff
@@ -65,6 +60,24 @@ export class FunctionEnvironmentTypeGenerator {
       EOL
     )}`;
 
-    fs.writeFileSync(this.typeDefFilePath, content);
+    this.writeShimFile(content);
   }
+
+  /**
+   * Generate an un-typed process.env shim
+   */
+  generateProcessEnvShim = () => {
+    const content = 'export const env = process.env;';
+    this.writeShimFile(content);
+  };
```

Comment:
At this point, the file should still have the headers saying it was generated by amplify and it's purpose. This will be helpful in case the synth fails and customers are looking at the file wondering where it came from.